models.js

```diff
@@ -0,0 +1,19 @@
+const db = require('./db/connection.js')
+
+function retreiveTopics(query) {
+
+
+
+    const sqlQuery = `SELECT * FROM topics;`
+
+ 
+    return db.query(sqlQuery).then((result) => {
+
+        if (result.rows.length === 0) {
```

Comment:
this is unlikely to ever happen, and for this ticket you probably don't need to bother trying to test this error, because you're always going to get some topics back